swarm/api/http/server_test.go

```diff
@@ -18,21 +18,78 @@ package http_test
 
 import (
 	"bytes"
+	"crypto/rand"
 	"errors"
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"os"
 	"strings"
 	"sync"
 	"testing"
 
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/swarm/api"
 	swarm "github.com/ethereum/go-ethereum/swarm/api/client"
 	"github.com/ethereum/go-ethereum/swarm/storage"
 	"github.com/ethereum/go-ethereum/swarm/testutil"
 )
 
+func init() {
+	log.Root().SetHandler(log.CallerFileHandler(log.LvlFilterHandler(log.LvlTrace, log.StreamHandler(os.Stderr, log.TerminalFormat(true)))))
```

Comment:
Is this necessary?

Reply:
While "in progress" - yes! ;)

Reply:
Let's add a loglevel command line flag, and then it is not disturbing at all, like here: https://github.com/ethersphere/go-ethereum/blob/swarm-network-rewrite-syncer/swarm/storage/common_test.go#L37

Reply:
Are we keeping this then?